src/Jsonize.Test/Program.cs

```diff
@@ -12,9 +12,11 @@ namespace Jsonize_Test
 {
     public class Program
     {
+        const string TEXTFILE = "WriteText.txt";
```

Comment:
Variables - when not specified - are by default `private`. 
But I can make them explicitly private.